src/Form/inputTypeConfig.js

```diff
@@ -169,7 +169,8 @@ const inputTypeConfig = {
         getIconFilled: getIdentitiesIcon,
         getIconAlternate: getIdentitiesAlternateIcon,
         shouldDecorate: async (input, { device }) => {
-            return canBeAutofilled(input, device);
+            // For iOS, we want to add handlers to the input to be able to show the keyboard extension
+            return device.globalConfig.isIOS || canBeAutofilled(input, device);
```

Comment:
This also feels like arm-twisting. I'm inclined to create a separate condition here … or actually, since this is not related to input types (we want to fire the focus on all types, so native knows to remove the accessory), we could have a dedicated logic on top of the `decorateInput` in Form.js. The idea is to de-couple the mobile focus logic (which is simpler) from the rest (which is very complex). The complex logic prevents the focus. The simpler logic is only concerned with sending the message on focus.